engines/python/setup/djl_python/rolling_batch/lmi_dist_v2_rolling_batch.py

```diff
@@ -79,6 +79,9 @@ def reset(self) -> None:
         self.request_cache = OrderedDict()
         super().reset()
 
+    def get_tokenizer(self):
+        return self.engine.tokenizer.tokenizer
```

Comment:
this has to be `self.engine.preprocessor.tokenizer`. `engine` does not have `tokenizer` attribute.

Reply:
@rohithkrn does the same applies to vLLM or Lmi-dist changed the location?

Reply:
no, this is specific to `lmi-dist`.